src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -728,7 +728,8 @@ public WebElement getStatusMessage() {
     public InstructorFeedbacksPage clickDoneEditingLink() {
         WebElement doneEditingLink = browser.driver.findElement(By.id("addNewQuestionTable"))
                                                    .findElements(By.tagName("a"))
-                                                   .get(3);
+                                                   .get(12);
```

Comment:
This is very fragile, although unrelated to your issue. Try spending a few minutes to see you can rewrite this to be more robust, otherwise, just make a new issue for this.